### PR TITLE
Merge "stable"

### PR DESCRIPTION
**Pull request by ilammy**
Following #728, merge `stable` into `master` to pull in latest updates.

The branches have diverged wildly, and there was cherry-picking left, right & center, and there was a merge of `release/0.13` not via a merge commit, etc. etc. etc. All this results in merge conflicts which need to be resolved manually. In ~~most~~ all cases the master version should be preferred.

## Checklist

- [X] ~~Change is covered by automated tests~~ (it is, but it will probably fail, wait for it to land)
- [X] Changelog is updated ~~(need see to it manually during the merge)~~
- [X] Resolve conflicts manually ~~via CLI~~ (GitHub now has an interface for that)
- [x] Merge manually via CLI after review

[coding guidelines]: https://github.com/cossacklabs/themis/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by ilammy** [comment]
Although GitHub now allows resolving conflicts via web interface, it's not *that* powerful to remember the resolution and apply it once the PR is merged. I've committed a merge with resolution to `stable` to show how it will be resolved. However, merging this as is into `master` will result in a "samba" merge which is not nice.

Once approved, I'll redo the merge manually and roll `stable` back to where it was.

File: gothemis/cell/cell.go
Comment (vixentael) [inline gothemis/cell/cell.go:286]: this is wrong, it's a duplication of the lines above, but using previous error name

**Review by vixentael — COMMENTED** [review]
I don't think that this is correct merge, I see changes on Go code that are present in master branch, not in stable branch.

I'm not sure what we should approve here, as the changes are wrong. Please merge manually and carefully.

**Comment by ilammy** [comment]
> I don't think that this is correct merge, I see changes on Go code that are present in master branch, not in stable branch.

Oh well... This is why I don't really like cherry-picking things all over the place. Since sometimes it ends up like this once you want to merge.

Okay, change of plans. Instead of letting GitHub commit the merge commit, I'll manually merge `stable` into `master`.

```
---o---o---o  master

---o---o---o  stable
```

This produces a merge commit:

```
---o---o---o  master
            \
             M
            /
---o---o---o  stable
```

I'll push that merge commit into `stable` to present it in this PR:

```
---o---o---o  master
            \
             \
              \
---o---o---o---M  stable
```

Once the review is complete, I'll push that commit as is into `master`:

```
---o---o---o---M  master, stable
              /
             /
            /
---o---o---o
```

and roll back `stable` back to where it was originally:


```
---o---o---o---M  master
              /
             /
            /
---o---o---o  stable
```

**Comment by ilammy** [comment]
Now, this one should be okay. I've resolved all conflicts it favor of `master`. There shouldn't be any changes at all to what we have in master currently. The merge will only tell git that any future merges from `stable` to `master` should apply only to new commits that will be made from this point.

**Comment by ilammy** [comment]
Thank you @vixentael for keeping me from doing silly things! :heart:

**Comment by vixentael** [comment]
thank you @ilammy! This PR changes look correct (as no changes from `stable` should be new to `master`, I think that empty changes is our goal).

**Review by vixentael — APPROVED** [review]

**Pull request merged**
